go-controller/pkg/cluster/bin/ovnkube-setup-master

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Fixed argument set
+
+API_TOKEN=$1
+K8S_API_SERVER_IP=${2-https://localhost:8443}
+MASTER_SWITCH_SUBNET=${3-11.11.1.0/24}
+CLUSTER_IP_SUBNET=${4-11.11.0.0/16}
+hostname=`hostname`
+NODE_NAME=${5-$hostname}
+
+if [[ "${API_TOKEN}" == "" ]]; then
+	echo "Supply kube access secret as the argument"
+	exit 1
+fi
+
+install() {
```

Comment:
This is very RHEL specific. Can we only call this when the platform supports yum?